pkgs/development/python-modules/surt/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, tldextract
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  version = "0.3.1";
+  pname = "surt";
+
+  src = fetchFromGitHub {
+    owner = "internetarchive";
+    repo = "surt";
+    rev = "6934c321b3e2f66af9c001d882475949f00570c5"; # Has no git tag
+    sha256 = "sha256-pSMNpFfq2V0ANWNFPcb1DwPHccbfddo9P4xZ+ghwbz4=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    tldextract
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-cov
+  ];
+
+  checkPhase = ''
```

Comment:
Why the custom checkPhase? You already have pytestCheckHook?

Reply:
I found the arguments to pytest in setup.py. If I remove the `checkPhase`, the output changes a little, being less verbose, not printing the coverage and also printing some warnings. But since all tests are still being executed and we don't really need any of the extra output anyways, I'll remove it and not even bother with `pytestFlagsArray` as suggested by @SuperSandro2000 below. This also removes the need for the extra `pytest-cov` dependency during testing.

Reply:
Ahhh, that makes sense. Yeah, I don't think generating the coverage makes much sense as part of a package build, and the verbose output isn't necessarily useful all the time either. I was mostly curious where it had come from :)